src/table/progressive-loading/progressive-loading-utils.ts

```diff
@@ -27,7 +27,7 @@ export function useProgressiveLoadingProps<T>({
     if (isItemExpanded(items[i]) && getItemChildren(items[i]).length === 0) {
       const status = getLoadingStatus?.(items[i]);
       if (status && (status === 'loading' || status === 'error')) {
-        allRows.push({ type: 'loader', item: items[i], level: getItemLevel(items[i]), status });
```

Comment:
can we have an explanation somewhere regarding the logic of assigning values to `from` ?

Reply:
Sure, will add a code comment